Storage/Controllers/AuthController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Google.Apis.Auth.AspNetCore3;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Drive.v3;
+using Google.Apis.Drive.v3.Data;
+using Google.Apis.Services;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Oauth2.v2;
+using Google.Apis.Oauth2.v2.Data;
+
+
+namespace Storage.Controllers
+{
+    [ApiController]
+    [Route("api/User")]
+    
+    public class AuthController : Controller
```

Comment:
```suggestion
    [Route("api/User")]
    public class AuthController : Controller
```
А ещё надо комментарии к каждому классу и каждому паблик-методу, если по-нормальному всё делать